grpc_utils/interceptor/request_id.go

```diff
@@ -0,0 +1,32 @@
+package interceptor
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/google/uuid"
+	app "github.com/oj-lab/go-webmods/app"
+	"google.golang.org/grpc"
+)
+
+type requestIDContextKey string
+
+const (
+	requestIDKey requestIDContextKey = "request_id"
+)
+
+// RequestIDInterceptor is a unary server interceptor that adds a request ID to the context and logs
+func RequestIDInterceptor(ctx context.Context, req any, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (any, error) {
+	requestID := uuid.New().String()
+	ctx = context.WithValue(ctx, requestIDKey, requestID)
```

Comment:
Remove this and related `requestIDContextKey` etc.
For the following reason:
1. We rarely need to get `request_id` in handlers and services.
2. Duplicate value stored in this context.